data/json/recipes/recipe_others.json

```diff
@@ -7904,5 +7904,36 @@
     [ [ "wheel", 1 ] ],
     [ [ "motor", 1 ] ]
   ]
+},{
+  "type" : "recipe",
+  "result": "neoprene",
+  "category": "CC_CHEM",
+  "subcategory": "CSC_CHEM_OTHER",
+  "skill_used": "cooking",
+  "skills_required": [ "cooking", 4 ],
+  "difficulty": 4,
```

Comment:
From a pure realism standpoint I agree. But considering that neoprene gear is kind of late early game gear it would make the gear itself outdated by the time you get it if you need 6 cooking. If you think realism should trump gameplay balance I'll change it. But consider that point.